### ISSUE TITLE
Danish audio track from swedish site version

### ISSUE THREAD

**Issue by fgustafsson** [emilsvennesson/kodi-viaplay#19]
As per the title, the Viaplay addon serves me the Danish audio track where it is supposed to serve a Swedish, e.g. children's shows. Subtitles behave as expected, as are English language shows doing. It is reliable across different titles.

Latest version, reinstalling does not change anything, nor changing addon preferences. Viaplay in browser does not behave this way.

I put on debug logs and checked them, but couldn't find anything relating to audio stream language, only site (viaplay.se in this case). Will of course upload if required.

Thank you!

**Comment by emilsvennesson** [comment]
Hi @fgustafsson!

You can set the default audio language in the Kodi settings. Player -> Language -> Preferred audio language

**Comment by fgustafsson** [comment]
.. and that probably only includes English, since I have not touched that setting. There will certainly not be any Danish involved.

Thank you, @emilsvennesson , I will check if that helps asap.

**Comment by fgustafsson** [comment]
Sorry for the late reply, Easter holidays got in the way. Indeed default audio language was set to "original stream" (or whatever that is called in English), and changing that to Swedish produced the desired results. You can close this if you like, but may I take the opportunity to suggest that unless default audio language is set (which is a feature for the addon, of course), the fallback should be the standard language of the selected site. That would probably be the expected desire for most users, while not getting in the way of customization options. E.g. When default audio is set to "Original stream" and Site to "viaplay.se", fall back to Swedish audio, when default audio is something else, respect that wish.

**Comment by emilsvennesson** [comment]
@fgustafsson Agreed, but this is easier said than done. Once the stream has been resolved from the add-on, it's basically in the hands of Kodi. The reason you can customize subtitles is because they're supplied externally and not included in the stream.

It might be possible to subclass `xbmc.Player` and change audio stream that way, but that's a bit of a workaround. I'm leaving this issue open for now because it would definitely be an enhancement.

**Comment by phalax** [comment]
Thank you for the System settings tip. This was a real annoyance. 😉

**Comment by woj76** [comment]
Same here (thanks!), I was about to report a bug when I had to cancel watching a kids movie because of this issue ;)
